robottelo/cli/base.py

```diff
@@ -86,6 +86,33 @@ class Base(object):
     logger = logging.getLogger('robottelo')
 
     @classmethod
+    def _handle_response(cls, response, ignore_stderr=None):
+        """Verifies ``return_code`` and ``stderr`` in ``SSHCommandResult``
+        response and throws ``CLICommandExecutionError`` exception if
+        ``return_code`` is not equal to 0.
+
+        """
+        if response.return_code != 0:
+            raise CLICommandExecutionError(
+                response.return_code,
+                response.stderr,
+                'Command \'{0} {1}\' finished with return_code {2}\n'
+                'stderr contains following message:\n{3}'
+                .format(
+                    cls.command_base,
+                    cls.command_sub,
+                    response.return_code,
+                    response.stderr,
+                )
+            )
+        if len(response.stderr) != 0 and not ignore_stderr:
+            cls.logger.warning(
```

Comment:
I am in favor of keeping the stderr in place. It can provide some useful information in the event of a failure. With that said, we should definitely continue to transition away from checking stderr as a test condition. Basically just use it in the vent of a failure, not as a reason to fail.